services/7segmentdisplay.md

```diff
@@ -0,0 +1,33 @@
+#  7-segment display
+
+    identifier: 0x196158f7
+
+A classic 7-segment display, with 4 dots and column.
+
+## Registers
+
+    rw digits @ value {
+    repeats:
+        digit: u8
+        dot: bool
+    }
+
+The digit and dots displayed on the segment. If ``digit`` is outside ``0x0-0xf``, nothing is displayed.
+
+    rw brightness: u8 @ intensity
+
+Controls the brightness of the LEDs. ``0`` means off.
+
+    rw column: bool @ 0x80
+
+Show or hide the column. If the column is not supported, the value remains false.
```

Comment:
what's column?

Reply:
some segments have double dot LED in the middle, to display time.